jkind-common/src/jkind/lustre/values/BooleanValue.java

```diff
@@ -26,13 +26,13 @@ public Value applyBinaryOp(BinaryOp op, Value right) {
 		if (right instanceof UnknownValue) {
 			switch (op) {
 			case OR:
-				return value ? fromBoolean(true) : UnknownValue.UNKNOWN;
+				return value ? BooleanValue.TRUE : UnknownValue.UNKNOWN;
```

Comment:
Since this code is in `BooleanValue` you should be able to say just `TRUE` instead of `BooleanValue.TRUE`. Same for other cases here.